kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -0,0 +1,25 @@
+const Kolibri = require('kolibri');
+
+/**
+ * Action to fetch a particular content node from the API.
+ * @param {Function} dispatch - The dispatch method of the store object.
+ * @param {String} id - The id of the model to be fetched.
+ */
+const fetchFullContent = ({ dispatch }, id) => {
+  // Get the model from the contentmetadata resource.
+  const contentModel = Kolibri.resources.getResource('contentmetadata').getModel(id);
```

Comment:
Could this be written as `ContentMetaDataResource.getModel(id)` and avoid hard-coding the URL string name?